client/src/pages/Profile.js

```diff
@@ -137,16 +143,10 @@ export default function ProfilePage() {
 
         {/* Right part*/}
         <Grid item xs={9} className={"flex-col-scroll"}>
-          <Typography
-            variant="h3"
-            gutterBottom
-            className={classes.projectTitle}
-          >
+          <Typography variant="h3" gutterBottom className={classes.projectTitle}>
             Projects
           </Typography>
-
-          <ProjectList projectData={projectData} />
-
+          <ProjectList projectData={projects} /> {/*the prop would be changed with projects state */}
```

Comment:
Is it working rendering the projects as expected?

Reply:
Currently, no projects are a part of a profile so that's why I wasnt able to test that out